tools/ansible/roles/dlpx.appliance-build-bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+
+#
+# This role configures a VM for building the Virtualization Engine.
```

Comment:
Update the comment to read "for building the Delphix Appliance"?

Reply:
I'd be in favor of just removing the comment...